docs/getting-started/tutorial-sample-query.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[tutorial-sample-query]]
+=== Querying the data
+
+You can also enter an {es} query to narrow the view of the data.  {kib} supports the 
+<<lucene-query>>.
+
+. In the query bar, enter this simple query:
++
```

Comment:
I'd probably flip this around as an example, "For example, to find all the flights out of Rome, submit the following query:"